web/packages/teleterm/src/ui/TopBar/Identity/IdentityList/IdentityList.tsx

```diff
@@ -103,7 +102,30 @@ export function ActiveCluster(props: {
             </Label>
           ))}
         </Flex>
-        <DeviceTrustMessage status={props.deviceTrustStatus} />
+        <Stack gap={0}>
+          {validUntil && (
+            <Flex gap={1} color="text.slightlyMuted">
+              <Clock size="small" />
+              <P3>
+                <span title={validUntil.toLocaleString()}>
+                  {isPast(validUntil)
+                    ? 'Session expired.'
+                    : `Session expires ${formatDistanceToNowStrict(validUntil, { addSuffix: true })}.`}
```

Comment:
I wasn't sure what's the best format to display the expiration date. I think our frontend code only uses `formatDistanceToNow`/`formatDistanceToNowStrict` to format a duration. But that format is less precise than the `tsh status` output. 
For example, in Connect you will get `Session expires in 10 hours` (with an exact date on hover), while tsh displays `valid for 10h11m0s`.

Do you have any thoughts on this?